modules/srst2/srst2/main.nf

```diff
@@ -0,0 +1,47 @@
+process SRST2_SRST2 {
+    tag "${meta.id}"
+    label 'process_low'
+
+    conda (params.enable_conda ? "bioconda::srst2=0.2.0" : null)
+    container "${ workflow.containerEngine == 'singularity' && !task.ext.singularity_pull_docker_container ?
+        'https://depot.galaxyproject.org/singularity/srst2%3A0.2.0--py27_2':
+        'quay.io/biocontainers/srst2:0.2.0--py27_2'}"
+
+    input:
+    tuple val(meta), path(fastq_s), path(db)
+
+    output:
+    tuple val(meta), path("*_genes_*_results.txt")               ,                emit: gene_results
+    tuple val(meta), path("*_fullgenes_*_results.txt")           , optional:true, emit: fullgene_results
+    tuple val(meta), path("*.pileup")                            ,                emit: pileup
+    tuple val(meta), path("*.sorted.bam")                        ,                emit: sorted_bam
+    path "versions.yml"                                          ,                emit: versions
+
+    when:
+    task.ext.when == null || task.ext.when
+
+    script:
+    def args = task.ext.args ?: ""
+    def prefix = task.ext.prefix ?: "${meta.id}"
+    def read_s = meta.single_end ? "--input_se ${fastq_s}" : "--input_pe ${fastq_s[0]} ${fastq_s[1]}"
+    if (meta.db=="gene") {
+        database = "--gene_db ${db}"
+    } else if (meta.db=="mlst") {
+        database = "--mlst_db ${db}"
+    } else {
+        return
```

Comment:
don't think you need to return here in groovy

Reply:
I was trying to get it to exit with an error. If you don't pick one of those two options nothing will happen so I was trying to get it to stop. Is there a better way to do it?

Reply:
do you have tests for the `mlst` db?